src/views/production/ProductionHeader.vue

```diff
@@ -83,27 +105,14 @@ export default {
   },
   computed: {
     venues() {
-      if (!this.production.performances.edges.length) return '';
-
-      let venues = [];
       if (this.hasInPersonPerformances) {
-        venues = lo.uniq(
+        return lo.uniqBy(
```

Comment:
We don't seem to be limiting the number of venues we are showing here.

Implement a way to limit the max we show, and we probably to to test at that

I.e. we should get

Live at various venues and online

When we go above 3 venues or something like that?